src/transforms/pipelines/config.rs

```diff
@@ -1,15 +1,25 @@
 use std::collections::HashSet;
 
-use indexmap::IndexMap;
 use serde::{Deserialize, Serialize};
-use vector_core::transform::{InnerTopology, InnerTopologyTransform};
+use vector_core::{
+    config::Input,
+    event::{Event, EventArray, EventContainer},
+    schema,
+    transform::{
+        InnerTopology, InnerTopologyTransform, SyncTransform, Transform, TransformContext,
+        TransformOutputsBuf,
+    },
+};
 
 use crate::{
-    conditions::AnyCondition,
+    conditions::{AnyCondition, Condition},
     config::{ComponentKey, DataType, Output, TransformConfig},
-    transforms::route::{RouteConfig, UNMATCHED_ROUTE},
 };
 
+// 32 is a lowish number and arbitrarily chosen: there is no magic to this magic
+// constant.
+const INTERIOR_BUFFER_SIZE: usize = 32;
```

Comment:
It has not. Figured we'd fiddle with this once soak results come in / results are considered reliable because the underlying implementation is good to go.